janusgraph-core/src/main/java/org/janusgraph/core/ConfiguredGraphFactory.java

```diff
@@ -139,6 +139,7 @@ public static JanusGraph close(String graphName) throws Exception {
         final JanusGraphManager jgm = JanusGraphManagerUtility.getInstance();
         Preconditions.checkNotNull(jgm, JANUS_GRAPH_MANAGER_EXPECTED_STATE_MSG);
         final Graph graph = jgm.removeGraph(graphName);
+        jgm.removeTraversalSource(graphName + "_traversal");
```

Comment:
might be a good idea to keep `_traversal` here and in line 249 below as a referenced constant.
Seems to imply usage of convention used in multiple places as a "magic string". In which case, changing in 1 place will break overall & dependent functions.
Also referenced [here](https://github.com/JanusGraph/janusgraph/blob/ac20052335506012e9b721b9897fef4b6100026e/janusgraph-core/src/main/java/org/janusgraph/graphdb/management/JanusGraphManager.java#L267) in the master branch.

Reply:
Agree with @angadsalaria .
We can also make a static method which converts `graphName` to Traversal Source name. Something like:
```
public static String toTraversalSourceName(String graphName){
    return graphName + "_traversal";
}
```
and then use that method to convert `graphName` to `traversalSourceName`.
I.e.:
```
jgm.removeTraversalSource(toTraversalSourceName(graphName));
```